src/sst/core/main.cc

```diff
@@ -368,36 +368,44 @@ main(int argc, char* argv[])
     }
     world_size.thread = cfg.getNumThreads();
 
+    // Create the factory.  This may be needed to load an external model definition
+    Factory* factory = new Factory(cfg.getLibPath());
+
+    // Get a list of all the available SSTModelDescriptions
+    std::vector<std::string> models = ELI::InfoDatabase::getRegisteredElementNames<SSTModelDescription>();
+
+    // Create a map of extensions to the model that supports them
+    std::map<std::string, std::string> extension_map;
+    for ( auto x : models ) {
+        auto extensions = factory->getSimpleInfo<SSTModelDescription, 1, std::vector<std::string>>(x);
+        for ( auto y : extensions ) {
+            extension_map[y] = x;
+        }
+    }
+
     SSTModelDescription* modelGen = nullptr;
 
     double start = sst_get_cpu_time();
 
     if ( cfg.configFile != "NONE" ) {
         string file_ext = "";
```

Comment:
It doesn't look like 'file_ext' is used anymore - can we remove this?

Reply:
Probably.  I'm about to do another clang-format PR, how about if I fix it there (unless something else pressing needs to be fixed in this PR)?

Reply:
@feldergast / @gvoskuilen - I went ahead and merged so that @feldergast can fix this up in the clang-format PR which is coming in due course.